src/SharedComponents/AstroVirtualAssistant/AstroVirtualAssistant.tsx

```diff
@@ -1,53 +1,95 @@
 import React, { FunctionComponent, useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
 
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 import { Stack, StackItem } from '@patternfly/react-core';
+import { useFlag } from '@unleash/proxy-client-react';
 
 import { Status, useAstro } from '../../Components/AstroChat/useAstro';
 import './astro-virtual-assistant.scss';
 import { AstroChat } from '../../Components/AstroChat/AstroChat';
 import { AstroBadge } from '../../Components/AstroAvatar/AstroBadge';
 import { commandMessageProcessor } from './CommandMessageProcessor';
-import { createPortal } from 'react-dom';
 
 const messageProcessors = [commandMessageProcessor];
 
-export const AstroVirtualAssistant: FunctionComponent = () => {
+interface AstroVirtualAssistantProps {
+  showAssistant: boolean;
```

Comment:
@justinorringer I'd love to see some tests local to the virtual-assistant frontend that eventually cover these props and expected behaviors.

Reply:
For context - I know we have tests in chrome - however it would be easy to regress something in this repo without those tests preventing a merge.